README.md

```diff
@@ -2,14 +2,26 @@ _NOTE: The interfaces of flux-accounting are being actively developed and are no
 
 ## flux-accounting
 
-Development for a bank/accounting interface for the Flux resource manager. Writes and saves job step history, user account information, and priority calculation to persistent storage using Python's SQLite3 API. 
+Development for a bank/accounting interface for the Flux resource manager. Writes and saves user account information to persistent storage using Python's SQLite3 API.
 
-##### job step history
+##### Install instructions
 
-Running as a cron job, `write_jobs.py` uses a flux-core RPC to retrieve job data that have recently transitioned to an INACTIVE state in the past _x_ minutes. It parses the job record data and writes it to a SQLite database, which can be queried at a later time. 
+This repository uses `pandas` for data display information. To install this dependency, along with any other packages specified in **requirements.txt**, and also run unit tests, you can run `make check`:
 
-##### user account information
+```
+$ make check
```

Comment:
I am not sure if I don't know if adding Makefile support for `make check` in an ad hoc fashion would be a long term solution. Did you add this as a short term stopgap?

@SteVwonder or @trws: is there any good python project that we can model after in terms of how to build, how to test and how to install etc?

Reply:
I think this python guide is a good data point, in particular the section on using a Makefile for kicking off the dependency installation and the test suite: https://docs.python-guide.org/writing/structure/#makefile

The python setuptools documentation mentions that the `test` command is deprecated and recommends the python package tox: https://setuptools.readthedocs.io/en/latest/setuptools.html#test-build-package-and-run-a-unittest-suite.  I think that tox is probably overkill right now for this project.

In terms of installation and distribution, `setup.py sdist` will create a source distribution that can be uploaded to PyPI.  `setup.py install` and `pip install .` will both install locally from the source tree.  `setup.py develop` and `pip install -e .` will install into develop mode (makes it look installed, but you can see edits to source files without having to re-install).  Relevant Docs:
- https://packaging.python.org/tutorials/installing-packages/
- https://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode

Reply:
Thanks. 

At first glance, it looks like autotools aren't best practices for python projects...

It will be a good test whether we can install our flux-accounting package properly with `flux-core` and `flux-sched` packages including support for both co-installation and standalone  installation) without GNU autotools and testing these configurations

`flux-sched` benefited heavily from mirroring various GNU autotools features.